Admin.js

```diff
@@ -114,561 +141,775 @@ var Admin = {
+    "show_form_first_tab_with_errors" (form, errorSelector) {
+
+        Admin.log("[core|show_form_first_tab_with_errors] show first tab with errors", form);
 
-        var tabs = form.find('.nav-tabs a'), firstTabWithErrors;
+        let tabs = form.find(".nav-tabs a"),
+            firstTabWithErrors;
 
-        tabs.each(function() {
-            var id = jQuery(this).attr('href'),
+        tabs.each(function () {
+
+            let id = jQuery(this).attr("href"),
                 tab = jQuery(this),
-                icon = tab.find('.has-errors');
+                icon = tab.find(".has-errors");
 
             if (jQuery(id).find(errorSelector).length > 0) {
+
                 // Only show first tab with errors
                 if (!firstTabWithErrors) {
-                    tab.tab('show');
+
+                    tab.tab("show");
                     firstTabWithErrors = tab;
+
                 }
 
-                icon.removeClass('hide');
+                icon.removeClass("hide");
+
             } else {
-                icon.addClass('hide');
+
+                icon.addClass("hide");
+
             }
+
         });
+
     },
 
-    setup_inline_form_errors: function(subject) {
-        Admin.log('[core|setup_inline_form_errors] show first tab with errors', subject);
+    "setup_inline_form_errors" (subject) {
+
+        Admin.log("[core|setup_inline_form_errors] show first tab with errors", subject);
 
-        var deleteCheckboxSelector = '.sonata-ba-field-inline-table [id$="_delete"][type="checkbox"]';
+        const deleteCheckboxSelector = ".sonata-ba-field-inline-table [id$=\"_delete\"][type=\"checkbox\"]";
+
+        jQuery(deleteCheckboxSelector, subject).each(function () {
 
-        jQuery(deleteCheckboxSelector, subject).each(function() {
             Admin.switch_inline_form_errors(jQuery(this));
+
         });
 
-        jQuery(subject).on('change', deleteCheckboxSelector, function() {
+        jQuery(subject).on("change", deleteCheckboxSelector, function () {
+
             Admin.switch_inline_form_errors(jQuery(this));
+
         });
+
     },
 
     /**
      * Disable inline form errors when the row is marked for deletion
      */
-    switch_inline_form_errors: function(subject) {
-        Admin.log('[core|switch_inline_form_errors] switch_inline_form_errors', subject);
+    "switch_inline_form_errors" (subject) {
 
-        var row = subject.closest('.sonata-ba-field-inline-table'),
-            errors = row.find('.sonata-ba-field-error-messages')
-        ;
+        Admin.log("[core|switch_inline_form_errors] switch_inline_form_errors", subject);
 
-        if (subject.is(':checked')) {
-            row
-                .find('[required]')
-                .removeAttr('required')
-                .attr('data-required', 'required')
-            ;
+        let row = subject.closest(".sonata-ba-field-inline-table"),
+            errors = row.find(".sonata-ba-field-error-messages");
```

Comment:
- eslint: Variables within the same declaration block should be sorted alphabetically.
- eslint: 'errors' is never reassigned. Use 'const' instead.